eclair-core/src/test/scala/fr/acinq/eclair/payment/receive/InvoicePurgerSpec.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair.payment.receive
+
+import akka.actor.typed.scaladsl.adapter.ClassicActorSystemOps
+import akka.testkit.TestProbe
+import fr.acinq.bitcoin.Block
+import fr.acinq.eclair.TestDatabases.TestPgDatabases
+import fr.acinq.eclair.db.{IncomingPayment, IncomingPaymentStatus, PaymentType, PaymentsDbSpec}
+import fr.acinq.eclair.payment.Bolt11Invoice
+import fr.acinq.eclair.payment.receive.InvoicePurger.{PurgeCompleted, PurgeEvent}
+import fr.acinq.eclair.{CltvExpiryDelta, MilliSatoshiLong, TestKitBaseClass, TimestampMilli, TimestampMilliLong, TimestampSecond, TimestampSecondLong, randomBytes32}
+import org.scalatest.funsuite.AnyFunSuiteLike
+
+import scala.concurrent.duration.DurationInt
+
+class InvoicePurgerSpec extends TestKitBaseClass with AnyFunSuiteLike {
+
+  import PaymentsDbSpec._
+
+  test("purge invoices") {
+    val dbs = TestPgDatabases()
```

Comment:
nit: I would use sqlite in tests, it's usually faster

Reply:
fixed in commit [853a46b](https://github.com/ACINQ/eclair/pull/2174/commits/853a46b760b54ccbbf4771565c3c2980f7cc74b9)